service/history/replication/task_processor_manager.go

```diff
@@ -52,7 +52,8 @@ import (
 )
 
 const (
-	clusterCallbackKey = "%s-%d" // <cluster name>-<polling shard id>
+	clusterCallbackKey   = "%s-%d" // <cluster name>-<polling shard id>
```

Comment:
is this key used anywhere? I can't find any reference to it.

Reply:
removed.